src/applications/resources-and-support/content-modeling.js

```diff
@@ -1,51 +0,0 @@
-/**
```

Comment:
It wasn't clear to me what `content-modeling` meant for this app, and this was just a constants mapping anyway, so I moved it to `constants`.